packages/playground/src/components/node_selector/TfDomainName.vue

```diff
@@ -52,7 +52,7 @@
               <template #append-item v-if="pagination.page !== -1">
                 <VContainer>
                   <VBtn
-                    @click="reloadDomains()"
+                    @click="loadDomains()"
```

Comment:
The `()` is not needed when calling a function without parameters in the template; it will be called normally.